talk/owt/sdk/base/win/shader.h

```diff
@@ -0,0 +1,2268 @@
+// Copyright (C) <2020> Intel Corporation

```

Comment:
Is this file borrowed from somewhere else? or any reference?

Reply:
This is compiled HLSL shader, not something borrowed elsewhere.